cmd/dep/vndr_importer.go

```diff
@@ -102,7 +102,8 @@ func (v *vndrImporter) convert(pr gps.ProjectRoot) (*dep.Manifest, *dep.Lock, er
 		}
 		pc.Constraint, err = v.sm.InferConstraint(pkg.revision, pc.Ident)
 		if err != nil {
-			return nil, nil, errors.Wrapf(err, "Unable to interpret revision specifier '%s' for package %s", pkg.importPath, pkg.revision)
+			v.logger.Printf("Unable to interpret revision specifier '%s' for package %s: %s", pkg.importPath, pkg.revision, err.Error())
+			continue
```

Comment:
FYI, `vndr` is a bit unique among the importers because it is always a revision being parsed, never a branch or a version. For glide or godep, they are converting a string that could represent a branch/version into a constraint. So while `continue` is okay here, we'll still need to add the project to the lock for the other importers when `InferConstraint` fails.